webapp-django/crashstats/crashstats/templatetags/jinja_helpers.py

```diff
@@ -244,8 +244,8 @@ def bugzilla_submit_url(report, bug_product):
         'product': bug_product,
         'op_sys': op_sys,
         'rep_platform': report['cpu_name'],
-        'cf_crash_signature': '[@ {}]'.format(report['signature']),
-        'short_desc': 'Crash in {}'.format(report['signature']),
+        'cf_crash_signature': u'[@ {}]'.format(report['signature']),
+        'short_desc': u'Crash in {}'.format(report['signature']),
```

Comment:
Hmm.. I wonder why did it work when it was a unicode string?
`urllib.urlencode` doesn't support an encoding. 

I'll just make them all byte strings. I think there's a Django function for it somewhere.